app/src/pages/authoring/index.tsx

```diff
@@ -65,6 +97,25 @@ export default function AuthoringPage() {
           <Button w={240} loading={draftMutation.isLoading} onClick={createResource}>
             {`Create New Draft ${resourceType}`}
           </Button>
+          <Title order={3}>{`Start From an Existing ${resourceType}:`}</Title>
+          {artifacts ? (
```

Comment:
I still think we should make use of the other attributes that we get back from a trpc `useQuery`. If there is an error or something, the user will just see a loading spinner with no info. A common pattern in trpc is something like this:

```tsx
const {
  data: artifacts,
  isLoading: artifactsLoading,
  error: artifactError
} = trpc.service.getArtifactsByResource.useQuery({ resourceType });


// ...

{artifactsLoading ? (
  <Loader />
) : artifactError ? (
  <div>{artifactError.message}</div>
) : artifacts ? (
  <Select
    label={`Select an existing ${resourceType} to create a draft from`}
    data={artifacts}
    value={selectedArtifact}
    onChange={setSelectedArtifact}
  />
) : (
  <div>An unknown error occurred fetching artifacts</div>
)}
```

You might want to do more styling than just a div for rendering the error message(s). The above is just to show the structure